src/components/MenuBar.vue

```diff
@@ -78,6 +86,7 @@ function logout() {
                 {{ user.email }}
               </p>
               <v-divider class="my-3"></v-divider>
+              <v-btn rounded variant="text" @click="redirectToLink(userURL)"> Profile </v-btn>
```

Comment:
One you have a router for the profile page created, you can remove the redirectTotLink function and just call something like
```
router.push({name: "profile" });
```